packages/ui/src/components/UnitInput/index.tsx

```diff
@@ -158,18 +166,21 @@ const UnitInputWrapper = styled(Stack)<{
     }
   }
 `
+const SelectInputWrapper = styled.div<{
+  width: number | string
+}>`
+${({ width }) => width && `width: ${typeof width === 'string' ? width : `${width}px`};`}
```

Comment:
This condition might now work. Let's say I set a width of "100%" which is an actual string it will do "100%px;". Maybe you meant if the width is a `number` ?